packages/shared/src/server/services/email/usageThresholdSuspension/UsageThresholdSuspensionEmailTemplate.tsx

```diff
@@ -0,0 +1,139 @@
+              <Text className="text-gray-700 text-sm leading-6">
+                Your organization &quot;{organizationName}&quot; has exceeded
+                the <strong>{limit.toLocaleString()} event limit</strong> for
+                the free tier. Data ingestion has been suspended.
+              </Text>
+            </Section>
+
+            <Section className="mt-8">
+              <div className="bg-red-50 border border-red-200 rounded-lg p-4 text-center">
+                <Text className="text-red-600 text-sm font-medium m-0 mb-1">
+                  Current Usage
+                </Text>
+                <Text className="text-3xl font-bold text-red-700 m-0">
+                  {currentUsage.toLocaleString()}
+                </Text>
+                <Text className="text-red-600 text-xs m-0 mt-1">
+                  Free tier limit: {limit.toLocaleString()} events
+                </Text>
+              </div>
+            </Section>
+
+            <Section className="mt-8 text-center">
+              <Button
+                className="rounded bg-red-600 px-5 py-3 text-center text-xs font-semibold text-white no-underline"
+                href={billingUrl}
+              >
+                Upgrade Now to Resume Ingestion
+              </Button>
+            </Section>
+
+            <Section className="mt-8">
+              <Heading className="text-black text-[18px] font-semibold">
+                What&apos;s affected?
+              </Heading>
+              <Text className="text-gray-700 text-sm leading-6">
+                •{" "}
+                <strong>
+                  New traces, observations, and scores cannot be ingested
+                </strong>
+                <br />
+                • Existing data remains accessible
+                <br />
+                • Dashboard and analytics continue to work
+                <br />• API calls to ingestion endpoints return 403 errors
+              </Text>
+            </Section>
+
+            <Section className="mt-8">
+              <Heading className="text-black text-[18px] font-semibold">
+                How to resolve:
+              </Heading>
+              <Text className="text-gray-700 text-sm leading-6">
+                • <strong>Upgrade plan</strong> for unlimited events and premium
+                features
+                <br />• Or contact support for custom plans and enterprise
+                options
+              </Text>
+            </Section>
+
+            <Hr className="border border-solid border-[#eaeaea] my-[26px] mx-0 w-full" />
+
+            <Section>
+              <Text className="text-[#666666] text-[12px] leading-[24px]">
+                This urgent notification was sent to {receiverEmail} regarding
+                ingestion suspension for &quot;{organizationName}&quot;.
+              </Text>
+              <Text className="text-[#666666] text-[12px] leading-[24px]">
+                Need immediate help? Contact us at{" "}
```

Comment:
we no longer want to push support@ contact option but rather prompt people to conatct us via the new support drawer. I think adding a "reply-to" to this email and ask people to reply if they have questions is a good idea -> thereby we at least have the context of this email